src/infection-model/rt.tsx

```diff
@@ -117,3 +118,11 @@ export const getRtDataForFacility = async (
     return null;
   }
 };
+
+export const getRtDataForFacilities = async (facilities: Facilities) => {
+  return await Promise.all(
+    facilities.map(async (facility) => {
+      return await getRtDataForFacility(facility);
+    }),
+  );
+};
```

Comment:
there's a similar function now on the MultiFacility page ... probably we should consolidate them into one (i think having it live here is probably fine)

Reply:
Sounds good...I'll try to update this so that we can retain the rtData caching/dispatching that's ont he multi facility dashboard